From a3899891e27fff4892e61a655d2db8005cbf75e2 Mon Sep 17 00:00:00 2001
From: Oak Zeng <Oak.Zeng@amd.com>
Date: Wed, 27 Sep 2017 18:21:21 -0400
Subject: [PATCH 5669/5855] drm/amdkfd: Fixed a unit convert error

schedule_delayed_work take jiffies as the second param
Convert ms to jiffies before call this func

Change-Id: Ied35d1b3703162fbfb5f39e9389ad3de8f89fac8
Signed-off-by: Oak Zeng <Oak.Zeng@amd.com>
---
 drivers/gpu/drm/amd/amdkfd/kfd_device.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/gpu/drm/amd/amdkfd/kfd_device.c b/drivers/gpu/drm/amd/amdkfd/kfd_device.c
index 88cb65e..60b774a 100644
--- a/drivers/gpu/drm/amd/amdkfd/kfd_device.c
+++ b/drivers/gpu/drm/amd/amdkfd/kfd_device.c
@@ -943,7 +943,7 @@ void kfd_restore_bo_worker(struct work_struct *work)
 		pr_info("Restore failed, try again after %d ms\n",
 			PROCESS_BACK_OFF_TIME_MS);
 		ret = schedule_delayed_work(&p->restore_work,
-				PROCESS_BACK_OFF_TIME_MS);
+				msecs_to_jiffies(PROCESS_BACK_OFF_TIME_MS));
 		WARN(!ret, "reschedule restore work failed\n");
 		return;
 	}
@@ -1043,7 +1043,7 @@ void kfd_evict_bo_worker(struct work_struct *work)
 	if (!ret) {
 		dma_fence_signal(eviction_work->quiesce_fence);
 		schedule_delayed_work(&p->restore_work,
-					PROCESS_RESTORE_TIME_MS);
+				msecs_to_jiffies(PROCESS_RESTORE_TIME_MS));
 	} else
 		pr_err("Failed to quiesce user queues. Cannot evict BOs\n");
 
-- 
2.7.4

